packages/webpack-extensive-lodash-replacement-plugin/index.js

```diff
@@ -67,7 +67,8 @@ class ExtensiveLodashReplacementPlugin {
 			throw createError( 'Could not determine root `lodash-es` version.' );
 		}
 
-		if ( baseLodashVersion !== this.baseLodashESVersion ) {
+		// TODO: change these to the exact versions when lodash and lodash-es releases match again
```

Comment:
This means we will be replacing `lodash@4.17.20` with `lodash-es@4.17.15`? I don't think that is valid, it will re-introduce bugs that lodash already fixed, including a security issue they fixed in v4.17.19. 

As an alternative approach we could drop this `lodash -> lodash-es` replacement, or change the plugin to replace in the other way (`lodash-es -> lodash`)

Reply:
I'm not saying it's a great approach 😉 Just trying to get to a temporary solution until `lodash-es` is released again. Happy to see alternatives be explored in the meantime, but I don't see this as something that's blocking us. How affected are we really by those 2 minor releases that were released in the meantime?

Reply:
I mean we don't even use `zipObjectDeep`

Reply:
What if tomorrow `lodash` releases `4.18.0`, with a ton of new features and somebody does the update? Everything will break because we force a replacement with `4.17.15`, and those errors may appear only in prod.

I think than the fastest and easiest alternative is just drop the lodash-es replacement, and deal with the hypothetical performance regression later.

Reply:
Well, the performance regression isn't that hypothetical, but I'll leave this call to @sgomes.

No strong feelings either way, but in general I believe this lodash problem should be solved separately and we shouldn't be blocking this PR.

Reply:
There's a lengthy discussion on the pros/cons of the change in Slack. But to summarise the summary:
- @tyxla's approach would preserve the current status quo while upgrading libraries
- Switching to `lodash` instead of `lodash-es` would make the critical path on small endpoints like `login` larger (bad) while making the total size of the application smaller (good, but not that impactful).

Dropping the plugin altogether is yet another option.